BackEnd/src/main/resources/application.yml

```diff
@@ -0,0 +1,17 @@
+spring:
+  application:
+    name: TradingApp
+  data:
+    mongodb:
+      uri: mongodb://localhost:27017/MarketStack
+  profiles:
+    active: default
+
+api:
+  closed-trade:
+    base-url: /api/v1/closedTrades
+    collection: Trade-Closed
+  open-trade:
+    base-url: /api/v1/openTrades
+    collection: Trade-Open
```

Comment:
You want to define base url paths here. Keeping everything in application.yml allows you to change things in _ONE_ place, rather than having to go search for where you may have hardcoded things. Try to avoid hardcoding as much as you can, if you want me to explain this one more lmk.